src/app/components/add-task/add-task.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {CUSTOM_ELEMENTS_SCHEMA} from '@angular/core';
+import {MatDialog} from '@angular/material/dialog';
+
+import {AddTaskComponent} from './add-task.component';
+
+describe('ButtonAddComponent', () => {
+  let component: AddTaskComponent;
+  let fixture: ComponentFixture<AddTaskComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [AddTaskComponent],
+      providers: [MatDialog],
```

Comment:
please import MatDialogModule instead of provide MatDialog service

Reply:
Done